src/components/AppNavigationItem/InputConfirmCancel.vue

```diff
@@ -0,0 +1,104 @@
+<template>
```

Comment:
Copyright header? :)

Reply:
I don't know much about copyright headers. I added one, but i have no idea if it is correct.
And should i mention @jancborchardt as a author ?